.travis.yml

```diff
@@ -23,11 +24,16 @@ matrix:
     - python: 3.6
       env: TOXENV=flake8
 
+before_install:
```

Comment:
From my reading here (https://docs.travis-ci.com/user/installing-dependencies) that seems to apply to container based infra and depends on a whitelist (linked above). Not sure if that whitelist refers to non-included-in-the-distribution, though.

The failure, however, seems to be related to the fact that the required package is not "dbus" I'll have to figure it out.